demos/python_demos/whiteboard_inpainting_demo/whiteboard_inpainting_demo.py

```diff
@@ -0,0 +1,185 @@
+    if args.m_instance_segmentation:
+        segmentation = MaskRCNN(ie, args.m_instance_segmentation, args.threshold,
+                                args.device, args.cpu_extension)
+    elif args.m_semantic_segmentation:
+        segmentation = SemanticSegmentation(ie, args.m_semantic_segmentation, args.threshold,
+                                            args.device, args.cpu_extension)
+
+    has_frame = True
+    output_frame = None
+    black_board = False
+    frame_number = 0
+    key = -1
+
+    while has_frame:
+        start = time.time()
+        if not args.no_show:
+            key = check_pressed_keys(key)
+            if key == 27:  # 'Esc'
+                break
+            elif key == ord('i'):  # catch pressing of key 'i'
+                black_board = not black_board
+                if output_frame is not None:
+                    output_frame = 255 - output_frame
+
+        has_frame, frame = capture.get_frame()
+        mask = None
+        if frame is not None:
+            detections = segmentation.get_detections([frame])
+            upsample_mask(detections)
+            if len(detections[0]) > 0:
+                mask = detections[0][0][2]
+                for i in range(1, len(detections[0])):
+                    mask = cv2.bitwise_or(mask, detections[0][i][2])
+
+        if mask is not None:
+            mask = np.stack([mask, mask, mask], axis=-1)
+        else:
+            mask = np.zeros(frame.shape, dtype='uint8')
+
+        clear_frame = remove_background(frame, invert_colors=not black_board)
+
+        if output_frame is None:
+            fill_value = 0 if black_board else 255
+            white_black = np.full(frame.shape, fill_value, dtype='uint8')
+            output_frame = np.where(mask, white_black, clear_frame)
+        else:
+            output_frame = np.where(mask, output_frame, clear_frame)
+        merged_frame = np.vstack([frame, output_frame])
+        merged_frame = cv2.resize(merged_frame, out_frame_size)
+
+        if output_video is not None:
+            output_video.write(merged_frame)
+        
+        if not args.no_show:
+            cv2.imshow(WINNAME, merged_frame)
+
+        if mouse.crop_available:
+            x0, x1 = min(mouse.points[0][0], mouse.points[1][0]), \
+                     max(mouse.points[0][0], mouse.points[1][0])
+            y0, y1 = min(mouse.points[0][1], mouse.points[1][1]), \
+                     max(mouse.points[0][1], mouse.points[1][1])
+            x1, y1 = min(x1, output_frame.shape[1] - 1), min(y1, output_frame.shape[0] - 1)
+            board = output_frame[y0: y1, x0: x1, :]
+            if board.shape[0] > 0 and board.shape[1] > 0:
+                cv2.namedWindow('Board', cv2.WINDOW_KEEPRATIO)
+                cv2.imshow('Board', board)
+            
+        end = time.time()
+        print('\rProcessing frame: {}, fps = {:.3}' \
+            .format(frame_number, 1. / (end - start)), end="")
+        frame_number += 1
+    
```

Comment:
Please, integrate `monitors.Presenter`: https://github.com/opencv/open_model_zoo/blob/e157f0ac4e98aa1e6ea21c56bc0bdc2d666d4cee/demos/python_demos/colorization_demo/colorization_demo.py#L150

Reply:
Done, but I did not realize how it works.

Reply:
Try `presenter = monitors.Presenter(args.utilization_monitors, 20, (out_frame_size[0] // 4, out_frame_size[1] // 16))` instead of current construction.

Reply:
Fixed.